### DISCUSSION TITLE
Feature Request: Footnote support

### DISCUSSION THREAD

**Discussion by selfire1** [tinacms/tinacms#4177] [Ideas]
Hi! A related issue, #566, was marked closed by stalebot. I'd just like to mention interest in supporting footnotes in the Tina Richtext editor. Writing a whole post in Tina only to edit it in another place to add footnotes is cumbersome.

In https://github.com/tinacms/tinacms/issues/3058#issuecomment-1232204672 it was mentioned that it is an aim to support footnotes internally. I'd love that functionality!

Related discussions are https://github.com/tinacms/tinacms/discussions/3140#discussioncomment-3512309 and https://github.com/tinacms/tinacms/issues/3058, which talk more about customising how the rich-text field behaves rather than adding internal footnote support.

**Comment by jaschaio** [comment]
I would love to see this as well - or even contribute to make it happen.

@selfire1 did you find any workaround in the meantime? The linked discussions and issue don't seem conclusive and footnotes is something pretty basic I would like to be able to implement.

**Reply by selfire1** [reply]
Unfortunately I didn't find a workaround. I ended up moving away from Tina 😬

**Reply by jaschaio** [reply]
Where did you move to?

**Reply by selfire1** [reply]
It was just for my personal blog, so I switched to a Markdown export from [Bear](https://bear.app/).

**Reply by scottgallant** [reply]
Love Bear!
